src/toil/resource.py

```diff
@@ -81,14 +95,15 @@ def create(cls, jobStore, leaderPath):
         with cls._load(leaderPath) as src:
             with jobStore.write_shared_file_stream(shared_file_name=pathHash, encrypted=False) as dst:
                 userScript = src.read()
-                contentHash.update(userScript)
-                dst.write(userScript)
+                if isinstance(userScript, bytes):
```

Comment:
I might do an `is not None` here to be a little clearer as to why it might not be a `bytes`.